frontend/compositions/dashboardHeader/desktopNav.tsx

```diff
@@ -16,9 +16,9 @@ export default function DesktopNav() {
 
   return (
     <ul className={styles.rightHeader} onClick={handleNavChange}>
-      <li className={setClassName('Search')} ><a href='#'>Search</a></li>
-      <li className={setClassName('Profile')} ><a href='#'>Profile</a></li>
-      <li className={setClassName('About')}><a href='#'>About</a></li>
+      <li tabIndex={0} className={setClassName('Search')} ><a href='#'>Search</a></li>
+      <li tabIndex={0} className={setClassName('Profile')} ><a href='#'>Profile</a></li>
+      <li tabIndex={0} className={setClassName('About')}><a href='#'>About</a></li>
```

Comment:
Why did we need to add tabIndex = 0? this will likely create a "double tab" block for keyboard users - we want them to be selecting & "clicking" directly on the link itself

Reply:
@srsexton94 seeing the double tab block with this unnecessary tab index - i've removed the tab indexes completely in my latest commit. Thanks for the double check on this.